test/byContract/ENSIntegration/ENSIntegration.ts

```diff
@@ -0,0 +1,128 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { deployments, ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    ENSRegistry,
+    ENSRegistry__factory,
+    IexecInterfaceNative,
+    IexecInterfaceNative__factory,
+    PublicResolver__factory,
+    ReverseRegistrar__factory,
+} from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+const CONFIG = require('../../../config/config.json');
+
+describe('ENSIntegration', async () => {
+    let proxyAddress: string;
+    let iexecPoco: IexecInterfaceNative;
+    let iexecPocoAsAdmin: IexecInterfaceNative;
+    let ensRegistry: ENSRegistry;
+    let iexecAdmin: SignerWithAddress;
+    let anyone: SignerWithAddress;
+
+    beforeEach('Deploy', async () => {
+        // Deploy all contracts
+        proxyAddress = await loadHardhatFixtureDeployment();
+        // Initialize test environment
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        ({ iexecAdmin, anyone } = await getIexecAccounts());
+        const ensRegistryAddress = (await deployments.get('ENSRegistry')).address;
+        ensRegistry = ENSRegistry__factory.connect(ensRegistryAddress, anyone);
+        iexecPoco = IexecInterfaceNative__factory.connect(proxyAddress, anyone);
+        iexecPocoAsAdmin = iexecPoco.connect(iexecAdmin);
+    }
+
+    describe('Forward resolution', async () => {
```

Comment:
I don't get the "forward" in the describe naming ?

Reply:
"Forward" resolution is the "normal" process of going from an ENS name to an address. We generally use "resolution" but we specify "forward" or "reverse" when both are in discussion.
See:
https://docs.ens.domains/learn/resolution#forward-resolution
https://docs.ens.domains/learn/resolution#reverse-resolution

Reply:
Thanks for the resources